examples/rust/fft/bin/main.rs

```diff
@@ -0,0 +1,53 @@
+use rustfft::{FftPlanner};
```

Comment:
Can you add a comment here that just says that this file is a test driver for the module, and is intended to be compiled and run on the native platform?

Reply:
I was going to ask you more about this. It's seems a bit inefficient to compile this with the module. This can be extended further to generate Makefile test (seems to be simple enough if we are going with the `make test` approach, I'm not sure). However, the main point is this can be kept separately, maybe a generate_tests library to generate tests for this (and potentially other modules we will be writing) but I don't know if it's a good idea or the best way to do this.

Reply:
Either way, I'm not sure it's possible to completely remove the cognitive overhead of having native tests (other than not to include them).